rotkehlchen/tests/unit/accounting/test_decoded_events.py

```diff
@@ -0,0 +1,200 @@
+# Some utility timestamps. They are used for mocked prices.
```

Comment:
Seems like the comment is off. And if it's for the tiemstamps after all imports also not needed

```suggestion
```